### PR TITLE
Bumping dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves multiple dependabot PRs.

**Overall change:** 
_Bumping dependencies._

**Code changes:**
- _Bump webpack-dev-server from 3.10.1 to 3.10.2 #5348_
- _Bump enzyme-to-json from 3.4.3 to 3.4.4 #5350_
- _Bump @testing-library/jest-dom from 5.0.2 to 5.1.0 #5354_
- _Bump jsdom from 16.0.1 to 16.1.0 #5355_
- _Bump @babel/runtime from 7.8.3 to 7.8.4 #5362_
- _Bump eslint-plugin-react from 7.17.0 to 7.18.2 #5366_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by HarveyPeachey — APPROVED** [review]
👍

**Review by j-pendlebury — CHANGES_REQUESTED** [review]
There are some `eslint` issues, which is why Jenkins is failing:
```
src/app/containers/ArticleMetadata/index.jsx
   6:3   error  'articleId' is missing in props validation               react/prop-types
  54:15  error  'linkedData' PropType is defined but prop is never used  react/no-unused-prop-types
```

**Comment by HarveyPeachey** [comment]
> There are some `eslint` issues, which is why Jenkins is failing:
> 
> ```
> src/app/containers/ArticleMetadata/index.jsx
>    6:3   error  'articleId' is missing in props validation               react/prop-types
>   54:15  error  'linkedData' PropType is defined but prop is never used  react/no-unused-prop-types
> ```

@jblaut Fahad's got a PR to fix that: https://github.com/bbc/simorgh/pull/5380

File: package.json [hunk 1]
```diff
@@ -170,15 +170,15 @@
     "eslint-plugin-json": "^2.0.1",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.2",
-    "eslint-plugin-react": "7.17.0",
+    "eslint-plugin-react": "7.18.2",
```
Comment (j-pendlebury) [inline package.json]:
```suggestion
    "eslint-plugin-react": "^7.18.2",
```

**Review by j-pendlebury — APPROVED** [review]
Just one suggestion, but otherwise looks good 👍

**Pull request merged**